app/views/idv/shared/_document_capture.html.erb

```diff
@@ -57,6 +57,12 @@
       <li><%= t('doc_auth.tips.document_capture_id_text4') %></li>
     </ul>
 
+    <%= f.input(
+      :document_capture_session_uuid,
+      as: :hidden,
+      input_html: { value: flow_session[:document_capture_session_uuid] },
```

Comment:
This isn't used by the lambdas or any callbacks, so I think it's ok to share this client-side, but wanted to confirm

Reply:
Yeah, we actually give it to the React component root element on L19 above.